bzrest/client.py

```
@@ -10,24 +10,42 @@
 
 
 class BugzillaClient(object):
-    def configure(self, bzurl, username, password):
+    def configure(self, bzurl, username=None, password=None, apikey=None):
+        if apikey:
+            self.apikey = apikey
+            self.username = None
+            self.password = None
+        elif username and password:
+            self.apikey = None
+            self.username = username
+            self.password = password
+        else:
+            raise ValueError("One of apikey or username & password must be supplied")
```

Accepting both (and silently throwing username/password away) makes for a confusing interface. I think it would be better to raise an error if username or password is passed in addition to apikey. Can you change that? Everything else looks fine, though I'd appreciate it if you tested it too :).